backend/lib/server_code/api/roomRoutes.js

```diff
@@ -14,24 +19,40 @@ roomRouter.use((req, res, next) => {
   next();
 });
 
-roomRouter.post("/create", (req, res) => {
+roomRouter.post("/create", async (req, res) => {
   const { creatorID } = req.body;
-  //roomPlayers.push(creatorID);
+  const roomID = roomIDCreator();
```

Comment:
the roomID is the roomIDCreator? I think you made a boo boo

Reply:
same here af259f1